src/routes/iscn/index.js

```diff
@@ -109,6 +112,20 @@ async function handleRegisterISCN(req, res, next) {
         broadcast: false,
       },
     );
+
+    if (req.query.estimate) {
+      const uploadPrice = req.uploadPrice || 0;
+      const iscnGasAndFee = await signingClient.esimateISCNTxGasAndFee(address, ISCNPayload);
+      // eslint-disable-next-line max-len
+      const changeISCNOwnershipNeed = new BigNumber(CHANGE_ISCN_OWNERSHIP_ESTIMATION_GAS).multipliedBy(DEFAULT_GAS_PRICE);
+      // eslint-disable-next-line max-len
+      const newISCNPrice = new BigNumber(iscnGasAndFee.gas.fee.amount[0].amount).plus(iscnGasAndFee.iscnFee.amount).plus(changeISCNOwnershipNeed).shiftedBy(-9)
+        .toNumber();
+      const totalNeed = newISCNPrice + uploadPrice;
+      res.json({ totalNeed });
```

Comment:
```suggestion
      res.json({ LIKE });
```
Call the result `LIKE` since its price in `LIKE`